src/led.h

```diff
@@ -15,4 +14,74 @@
 #define LED_h   0x12
 #define LED_dp  0x13
 
-void filldisplay(uint8_t pos, uint8_t val, __bit dp);
+const uint8_t __at (0x1000) ledtable[]
```

Comment:
I was able to decorate/relocate a function, e.g.
```
uint8_t __at (0x1000) foobar(uint8_t n) {
...
}
```
And call it from main section of code.
Also, stcgal, throws a warning, but the eeprom section of code is programmed, and it is working ;)